src/configure/helper/templateHelper.ts

```diff
@@ -176,45 +178,52 @@ export async function getFilteredTemplates(azureSession: AzureSession, resourceT
     let filteredTemplates: TemplateInfo[];
     switch (resourceType) {
         case TargetResourceType.AKS:
-            filteredTemplates = await client.getTemplatesInfoByFilter("docker", "Azure:AKS");
+            await telemetryHelper.executeFunctionWithTimeTelemetry(async () => {
```

Comment:
we should use the defined function `getTemplates` here as well

Reply:
getTemplates get templates based on repoAnlaysis result and this function is to get templates filtered on language, buildTarget and deployTarget.
Why should getTemplates be called here?

Reply:
yes,you are right